raiden-dapp/src/services/raiden-service.ts

```diff
@@ -88,6 +90,10 @@ export default class RaidenService {
     return this._userDepositTokenAddress;
   }
 
+  get monitoringReward(): BigNumber | null {
```

Comment:
not important but maybe ignore this in coverage?

Reply:
Already added it locally, just waiting to see if codecov just hangs or if it's already sufficient coverage 😅